tinymce-basics/useful_styles_for_content_css.html

```diff
@@ -0,0 +1,153 @@
+            // rounded corners on images example
+            // https://www.tiny.cloud/docs/configure/advanced-editing-behavior/#object_resizing
+            object_resizing : false,
+
+            // The following css will be injected into the editor, extending
+            // the default styles.
+            // In a real world scenario, with much more custom styles for headings
+            // links, tables, images etc, it's strongly recommended to use the
+            // content_css option to load a separate CSS file. Makes editing easier too.
+            // https://www.tiny.cloud/docs/configure/content-appearance/#content_style
+            // https://www.tiny.cloud/docs/configure/content-appearance/#content_css
+            content_style: `
+
+                /*
+                 * -----------------------------------------------------
+                 * Improve horizontal rule clickability
+                 * -----------------------------------------------------
+                 */
+
+                /* Make our invisible overlay be positioned relative to the hr */
+                hr {
+                  position: relative;
+                }
+
+                /* Create an invisible overlay that greatly increases the clickable area */
+                hr::after {
+                  content: '';
+                  display: block;
+                  height: 21px;
+                  left: 0;
+                  position: absolute;
+                  right: 0;
+                  top: -10px;
+                }
+
+                /*
+                 * -----------------------------------------------------
+                 * Paragraphs in table cells
+                 * -----------------------------------------------------
+                 */
+
+                /* Remove all margins from elements that are direct children
+                   of td and th. Here we are using the universal selector but
+                   to give more control you could narrow down to the specific
+                   selectors you support in your tables */
+                td > *,
+                th > * {
+                    margin: 0;
+                }
+
+                /* Use the famous lobotomized owl selector to reintroduce a margin
+                   in-between elements inside table cells. */
+                td > * + *,
+                th > * + * {
+                    margin-top: .75em;
+                }
+
+                /*
+                 * -----------------------------------------------------
+                 * Use box shadow instead of outline for selectable elements
+                 * -----------------------------------------------------
+                 */
+
+                img {
+                    border-radius: 10px;
+                }
+
+                /* Override the default content styles to remove the outline
+                   and reintroduce it as a box-shadow */
+                .mce-content-body img[data-mce-selected] {
```

Comment:
It would probably be good to clarify this is a style that applies only while editing content withing TinyMCE? As this isn't something you'd normally want in a content css file, since it won't be applicable on your own site.